int64 map_id
---
int64 retval
